packages/domain/src/use-cases/profile/FollowProfile.ts

```diff
@@ -17,19 +20,21 @@ import {
 
 export type FollowRequestFee = {
   amount: Amount<Erc20>;
-  contractAddress: string;
+  contractAddress: EvmAddress;
   recipient: EvmAddress;
 };
 
 export type UnconstrainedFollowRequest = {
   profileId: ProfileId;
   kind: TransactionKind.FOLLOW_PROFILE;
+  delegate: boolean;
 };
 
 export type PaidFollowRequest = {
   profileId: ProfileId;
   kind: TransactionKind.FOLLOW_PROFILE;
   fee: FollowRequestFee;
+  delegate: boolean;
```

Comment:
This one reads odd... "paid operations" cannot be delegated to our Lens Manager.

Reply:
I think I understand. Experience similar issue with open action. Let me push how I solved there in a second so you can fix it the same way.

Reply:
It was an issue in the `DelegableSigning<T>` typedef. "signed requests" is a superset of "delegable requests" for a given operation.

Reply:
the issue was that TS is complaining when both types in a union don't have `delegate` field. how to solve it?